docs-ui/stories/components/sidebarSection.stories.js

```diff
@@ -0,0 +1,92 @@
+import styled from '@emotion/styled';
+
+import {KeyValueTable, KeyValueTableRow} from 'app/components/keyValueTable';
+import {Body, Main, Side} from 'app/components/layouts/thirds';
+import QuestionTooltip from 'app/components/questionTooltip';
+import SidebarSection from 'app/components/sidebarSection';
+import Tag from 'app/components/tag';
+import TextOverflow from 'app/components/textOverflow';
+import Tooltip from 'app/components/tooltip';
+import space from 'app/styles/space';
+
+export default {
+  title: 'Views/Sidebar Section',
+};
+
+export const Default = () => (
+  <Body>
+    <Main />
+    <Side>
```

Comment:
I'm not sure if this is needed if the intent of this story is to show off how to use `<SidebarSection/>` -- although you can make the case that this component should *ONLY* be used on the right side? (If so I would explicitly state that).

Reply:
Sounds good, I was mostly trying to show how it would look on the side of a page, but it can go on either side - I'll remove these